src/pudl/transform/ferc1.py

```diff
@@ -716,6 +716,48 @@ def combine_axis_columns_xbrl(
     return df
 
 
+class AssignQuarterlyFiledDataToAnnualDbf(TransformParams):
+    """Parameters for transfering quarterly reported data to annual columns."""
+
+    annual_cols: list[str] = []
+    quarterly_cols: list[str] = []
```

Comment:
There's an implicit 1:1 linkage between the individual elements of these two lists.  If they are in different orders such that `annual_cols[n]` does not correspond to `quarterly_cols[n]` then the result of the transformation will be incorrect.  A data structure that explicitly ties the quarterly and annual columns to each other feels like it would be more robust, like a dictionary that maps individual quarterly columns to their corresponding annual columns, and would also mean you don't need to validate that you've got the same number of columns in each attribute.

Reply:
this is a good idea I'll convert these two guys to a dict